frameworks/Python/bottle/uwsgi.ini

```diff
@@ -2,7 +2,7 @@
 master
 ; Increase listen queue used for nginx connecting to uWSGI. This matches
 ; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
-listen = 65535
+listen = 128
```

Comment:
this should be adjusted to variable from sysctl net.core.somaxconn and should be much higher than 128.